### PR TITLE
new question with tests

### PR DESCRIPTION
**Pull request by alexrpedersen**
with correct solution all 5 test pass

### CODE REVIEW

File: test/array-tests.js [hunk 1]
```diff
@@ -233,4 +233,56 @@ describe('Array', function() {
...
+    });
+
+    it('should return null', function() {
+      let input = [];
+      let expectedResults = [];
+
+      let results = arrayQuestions.removeDuplicateIntegers(input)
+
+      assert.strictEqual(results.length, 0);
+      assert.deepStrictEqual(results, expectedResults);
```
Comment (Nardax) [inline test/array-tests.js]: duplicate of the above assert

File: test/array-tests.js [hunk 2]
```diff
@@ -233,4 +233,56 @@ describe('Array', function() {
...
+      let input = [2, 2, 2, 2];
+      let expectedResults = [2];
+
+      let results = arrayQuestions.removeDuplicateIntegers(input)
+
+      assert.strictEqual(results.length, 1);
+      assert.deepStrictEqual(results, expectedResults);
+    });
+
+    it('should return null', function() {
```
Comment (Nardax) [inline test/array-tests.js]: Should return empty array

File: test/array-tests.js [hunk 3]
```diff
@@ -233,4 +233,56 @@ describe('Array', function() {
       assert.strictEqual(results.length, 0);
     });
   });
+
+  describe('#removeDuplicateIntegers()', function(){
+    it('should return array without duplicate', function() {
+      let input = [1,2,3,3,4,5];
```
Comment (Nardax) [inline test/array-tests.js]: add spaces in array to match everything else

**Review by Nardax — CHANGES_REQUESTED** [review]
missing a test for passing in null

File: test/array-tests.js (hunk 1 above)
Comment (alexrpedersen) [reply test/array-tests.js]: I dont understand this comment

Comment (alexrpedersen) [reply test/array-tests.js]: got it
